.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,13 +1,16 @@
 <component name="InspectionProjectProfileManager">
   <profile version="1.0">
     <option name="myName" value="Project Default" />
+    <inspection_tool class="AutoCloseableResource" enabled="true" level="WARNING" enabled_by_default="true">
```

Comment:
This is an IntelliJ auto-updated file. I'm actually kind of surprised we have these committed.